liquid.py

```diff
@@ -204,6 +204,14 @@ def shell(name, *args):
     docker_exec_cmd = ['docker', 'exec', '-it', id] + list(args or ['bash'])
     run_fg(docker_exec_cmd, shell=False)
 
+def usershell(user, name, *args):
+    """
+    Open a shell in a docker container tagged with liquid_task=`name` as 'user'
```

Comment:
Ok, why don't we add a flag to the `shell` command? It does receive all arguments in `args` and `argparse` is good at parsing them. Something like:

```python
import argparse
parser = argparse.ArgumentParser()
parser.add_argument('-u', '--username')
parser.add_argument('cmd', nargs=argparse.REMAINDER)
options = parser.parse_args(args)
```

This creates a potential to capture `-u` flags from the command, but a user can work around that:

```shell
./liquid.py shell -- do_something.sh -u other  # `-u` is passed on to `do_something.sh`
```